### DISCUSSION TITLE
RelationManager doesn't load on production

### DISCUSSION THREAD

**Discussion by robertosanval** [filamentphp/filament#7812] [Help] [answered]
Hello, I have a model Resource called Document with related Lines as a HasMany relation. I've created a LinesRelationManager that works perfectly in local development but not on a production server.

I've checked errors on logs and there's nothing about this.

Any ideas?

**Comment by pboivin** [comment]
What is not working exactly? Can you describe the issue?

**Reply by robertosanval** [reply]
Related records doesn't appear

**Comment by robertosanval** [comment]
I've noticed that the parameter activeRelationManager doesn't exists on the URL on production

**Comment by robertosanval** [comment, marked as answer]
I found the problem.

I was using [bezhansalleh/filament-shield](https://github.com/bezhanSalleh/filament-shield) and I accidentally removed the access to that resource.
